[2024-05-14T01:54:14.953+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-14T01:54:14.969+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline_jobs104.say_hello manual__2024-05-14T01:12:13.781683+00:00 [queued]>
[2024-05-14T01:54:14.974+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline_jobs104.say_hello manual__2024-05-14T01:12:13.781683+00:00 [queued]>
[2024-05-14T01:54:14.974+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-14T01:54:14.981+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): say_hello> on 2024-05-14 01:12:13.781683+00:00
[2024-05-14T01:54:14.995+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=49) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-14T01:54:15.001+0000] {standard_task_runner.py:63} INFO - Started process 51 to run task
[2024-05-14T01:54:15.000+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline_jobs104', 'say_hello', 'manual__2024-05-14T01:12:13.781683+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_jobs104.py', '--cfg-path', '/tmp/tmpu_9g3a4x']
[2024-05-14T01:54:15.006+0000] {standard_task_runner.py:91} INFO - Job 5: Subtask say_hello
[2024-05-14T01:54:15.068+0000] {task_command.py:426} INFO - Running <TaskInstance: data_pipeline_jobs104.say_hello manual__2024-05-14T01:12:13.781683+00:00 [running]> on host ccd214b2d6c3
[2024-05-14T01:54:15.130+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline_jobs104' AIRFLOW_CTX_TASK_ID='say_hello' AIRFLOW_CTX_EXECUTION_DATE='2024-05-14T01:12:13.781683+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-14T01:12:13.781683+00:00'
[2024-05-14T01:54:15.132+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-14T01:54:15.133+0000] {logging_mixin.py:188} INFO - hello, yes
[2024-05-14T01:54:15.133+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-14T01:54:15.133+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-14T01:54:15.138+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_pipeline_jobs104, task_id=say_hello, run_id=manual__2024-05-14T01:12:13.781683+00:00, execution_date=20240514T011213, start_date=20240514T015414, end_date=20240514T015415
[2024-05-14T01:54:15.187+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-14T01:54:15.196+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-14T01:54:15.197+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
